pkg/controller/servicemesh/memberroll/controller.go

```diff
@@ -831,9 +848,23 @@ func (r *namespaceReconciler) addNetworkAttachmentDefinition(namespace, meshName
 	}
 
 	// TODO: update resource if its state isn't what we want
-
 	netAttachDef.SetNamespace(namespace)
 	netAttachDef.SetName(name)
+	config := &runtimeCNIConfig{
```

Comment:
Should this config match what's in the config map?  If so, maybe we should read the config out of the config map and use that instead of hard coding it here.

Reply:
I don't think it would work in a single daemonset case (we create it once, but need separate cni configs for each mesh/cni version).